### ISSUE TITLE
the valset of rad

### ISSUE THREAD

**Issue by GCKt** [aioz-ai/MICCAI21_MMQ#12]
Hello, I recently read the MMQ paper and code, there is a question, that is, the RAD dataset does not have a validation set, then I want to ask, what do we rely on to find the best accuracy model, and then use the test set to test the final accuracy? Thank you very much for your answer.

**Comment by zhangtianmingxp** [comment]
Hello, we also encountered the same problem, that is, after training 40 epochs, how do we choose the best model to test the accuracy on the test set? Is it to load all the generated weight models of 40 epochs in order to test the accuracy in the test set? Then choose the highest precision as the best result?

**Comment by zhangtianmingxp** [comment]
Thank you very much for your reply
